awx/main/tasks/jobs.py

```diff
@@ -622,21 +458,21 @@ def run(self, pk, **kwargs):
         status, rc = 'error', None
         extra_update_fields = {}
         fact_modification_times = {}
-        self.event_ct = 0
+        self.runner_callback.event_ct = 0
```

Comment:
I think it would be best to move this to the `__init__` of the callback class, since it's just setting it to 0. That will also help with cases that we later reference `runner_callback.event_ct`, prevent attribute errors if this hasn't happened yet.

Reply:
Done , will be pushed in the next commit